app/views/steps/case_details/edit.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= javascript_import_module_tag 'offence_autocomplete' %>
```

Comment:
it's a good standard to follow I guess but technically inconsequential because the JS code I have is built to only run when the DOM content is loaded. Regardless will update it for extra safety